src/VisualStudio/Core/Def/Implementation/ProjectSystem/Legacy/AbstractLegacyProject.cs

```diff
@@ -64,7 +64,7 @@ public override void Disconnect()
         protected void AddFile(string filename, SourceCodeKind sourceCodeKind)
         {
             bool getIsCurrentContext(IVisualStudioHostDocument document) => LinkedFileUtilities.IsCurrentContextHierarchy(document, RunningDocumentTable);
```

Comment:
this call path from legacy project system (ICSProject... ), I believe you guys not using? didn't want to change IDL to add this new API so, leaving it as it is...

Reply:
Razor for .NET Core is entirely CPS-based, which is the area we're interested in improving.

Reply:
okay. so we don't need it here for now.